src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+class Circle implements Figure {
```

Comment:
The class name 'Circle' should not start with a verb. Consider renaming it to something more indicative of what it represents, like 'CircularShape' or similar.